pkg/ccl/backupccl/restore_online_test.go

```diff
@@ -57,6 +60,12 @@ func TestOnlineRestoreBasic(t *testing.T) {
 
 	sqlDB.Exec(t, fmt.Sprintf("BACKUP INTO '%s'", externalStorage))
 
+	if rng.Float64() < 0.5 {
+		sqlDB.Exec(t, "SET sql_safe_updates = false;")
```

Comment:
rather than set this for the whole session maybe just add `where true` to the update